package.json

```diff
@@ -41,7 +41,7 @@
     "bottleneck": "1.5.x",
     "event-stream": "3.2.2",
     "express": "4.11.2",
-    "hiredis": "0.4.0",
+    "hiredis": "~0.4.1",
```

Comment:
OK, do you still want that change in this PR? 
saw the hiredis-node comment on gcc.